ppr-ui/src/components/common/Breadcrumb.vue

```diff
@@ -127,6 +127,7 @@ export default defineComponent({
     const handleStaff = (breadcrumbText): string => {
       if (localState.isStaff) {
         breadcrumbText = breadcrumbText.replace('My', 'Staff')
+        if (breadcrumbText === 'BC Registries Dashboard') breadcrumbText = 'Staff Dashboard'
```

Comment:
This is not the best way to handle it. It is not future-proof in case the text gets updated. Is there a way to rework it?

Reply:
For sure, agreed we shouldn't use a string here. 
Perhaps you could define the `Staff Dashboard` as a const and replace `tombstoneBreadcrumbDashboard[1].text` or redefine it etc vs a string swap/replace?